test/unit/polymer.element.html

```diff
@@ -80,6 +80,11 @@
     this._calledReady++;
   }
 
+  _attachDom(dom) {
+    this.appendChild(dom);
+    return this;
+  }
```

Comment:
Was this change necessary, or is it demonstrating usage?

With this change, the test element doesn't have a shadow root I think. Could we assert for that in this test?

Reply:
Not necessary. In the Contributor docs, it says every change has to be accompanied by some change in tests.

I'll try adding an assert.